src/main/java/com/kodilla/ecommercee/CartController.java

```diff
@@ -1,4 +1,44 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.domain.CartDto;
+import com.kodilla.ecommercee.mapper.CartMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/V1/cart")
 public class CartController {
+    @Autowired
+    private CartMapper cartMapper;
```

Comment:
wstrzykiwanie przez konstruktor dziala, ale opcjonalnie lepsza praktyka jest wstrzykiwanie przez konstruktor 
https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-spring-beans-and-dependency-injection.html